onnxscript/ir/_external_data.py

```diff
@@ -270,6 +270,12 @@ def convert_tensors_to_external(
         external_tensors[i]
         for i in sorted(range(len(external_tensors)), key=lambda i: sorted_indices[i])
     ]
+
+    # Clean-up temporary file if it is created
+    tmp_path = os.path.join(base_path, "tmp", relative_path)
+    if os.path.exists(tmp_path):
+        os.remove(tmp_path)
+
```

Comment:
Consider set a flag at 243 and do this only when the flag is set? Otherwise if there already exists a file named tep*** this will remove it unexpectedly.

Reply:
Agree. Also, can't we use `tempfile`?

Reply:
Had avoided using tempfile to avoid copy data between files, so a renaming should be sufficient here?